docs/building-blocks/extending-templates/conditional-linking.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Extending Azure Resource Manager Template functionality - Conditional Template Linking
+description: Describes how to extend the functionality of Azure Resource Manager templates to implement conditional template linking
+author: petertay
+ms.service: guidance
```

Comment:
You can omit ms.service, ms.topic, and ms.author. It's handled by the global metadata file now.